### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by Vipinofficial11**

### CODE REVIEW

File: .github/workflows/trigger.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ on:
 
 jobs:
   trigger:
-    runs-on: ubuntu-latest
+    runs-on: k8s-runner-e2e
```
Comment (itsankit-google) [inline .github/workflows/trigger.yml]: this workflow does not need to run on gke runner.
Please revert this change.

Comment (Vipinofficial11) [reply .github/workflows/trigger.yml]: reverted.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -15,75 +15,86 @@
...
-          
+          path: ./plugin/target/cucumber-reports
+          destination: e2e-tests-cucumber-reports/${{ github.event.repository.name }}/${{ github.ref }}
+
       - name: github-status-action
         uses: Sibz/github-status-action@67af1f4042a5a790681aad83c44008ca6cfab83d
         if: always()
         with:
-          authToken: ${{ secrets.GITHUB_TOKEN }}
+          authToken: ${{ secrets.AUTH_TOKEN }}
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml]: why is this change needed?

File: .github/workflows/e2e.yml [hunk 3]
```diff
@@ -15,75 +15,86 @@
...
-    if: >
-      github.event_name == 'workflow_dispatch'
-      || github.event_name == 'push'
-      || (contains(github.event.pull_request.labels.*.name, 'build')
-         && (github.event.action != 'labeled' || github.event.label.name == 'build')
-         )
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+
     steps:
-      # Pinned 1.0.0 version
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml:43]: pls revert this change

File: .github/workflows/e2e.yml (hunk 2 above)
Comment (Vipinofficial11) [reply .github/workflows/e2e.yml]: reverted the change (18bd6bbba7443d9e5c842d005d05000f3f295928)

File: .github/workflows/e2e.yml (hunk 3 above)
Comment (Vipinofficial11) [reply .github/workflows/e2e.yml:43]: Added the comment back (18bd6bbba7443d9e5c842d005d05000f3f295928)

**Review by itsankit-google — APPROVED** [review]
Please squash commits.

**Comment by Vipinofficial11** [comment]
> Please squash commits.

Commits Squashed.
